### PR TITLE
final reviewed code

### PR DESCRIPTION
**Pull request by abijithanikkuruthi**

### CODE REVIEW

File: src/spark.py [hunk 1]
```diff
@@ -17,27 +17,25 @@
...
-if args.kb:
-    kb_path = args.kb
 if args.f:
     input_path = args.f
 if args.hdfsout:
     output_path = args.hdfsout
 if args.debug == "True":
     debug = True
     model_root_path = "/data"
+    #kb_root_path = "/trident/data"
```
Comment (l8518) [inline src/spark.py]: remove?

File: src/WARCSplitReader.py [hunk 2]
```diff
@@ -4,8 +4,8 @@
 import lxml.html as lh
 from lxml import etree
 import bs4
-from config import TMP_FOLDER, WARC_ID, WARC_PER_DOC
 
+WARC_ID = "WARC-TREC-ID"
```
Comment (l8518) [inline src/WARCSplitReader.py]: is this correct? should be controlable via env or arg right?

File: setup.sh [hunk 3]
```diff
@@ -8,20 +8,34 @@ if [ $? -eq 0 ]; then echo "All done loading hadoop"; else echo "Error in loadin
 module load python/3.6.0    # Load python 3.6.0
 if [ $? -eq 0 ]; then echo "All done loading python3.6"; else echo "Error in loading Python" && exit 1; fi
 
-if [ ! -d "/home/wdps1936/elasticsearch-2.4.1" ]; then
+SCRATCH_PATH='/var/scratch2/wdps1936/'
```
Comment (l8518) [inline setup.sh]: sure this will work all the time?

File: src/TextExtraction.py [hunk 4]
```diff
@@ -93,8 +93,8 @@ def extract(row, extractor):
 
             row["text"] = text
 
-            # Split text into different sentences
-            row["sentences"] = row["text"].split("\n")
+            # Split text into different paragraphs
+            row["paragraphs"] = row["text"].split("\n")
```
Comment (l8518) [inline src/TextExtraction.py]: 👍

File: src/spark.py [hunk 5]
```diff
@@ -69,37 +60,24 @@
 txtprepro_stage_rdd.cache()
 print("Processed: {0}".format(txtprepro_stage_rdd.count()))
 
-if debug:
-    for row in txtprepro_stage_rdd.collect():
-        print(row["sentences"])
-
-# TODO: Currently Limiting: REMOVE LATER!
```
Comment (l8518) [inline src/spark.py]: 👍

**Comment by l8518** [comment]
@abijithanikkuruthi Great work 🔥

File: setup.sh (hunk 3 above)
Comment (abijithanikkuruthi) [reply setup.sh]: Yeah! I am creating a directory in the next line (mkdir -p $SCRATCH_PATH) in case the wdps1936 folder is absent.

Comment (l8518) [reply setup.sh]: oh 😅 🤦‍♂

File: src/WARCSplitReader.py (hunk 2 above)
Comment (l8518) [reply src/WARCSplitReader.py]: @abijithanikkuruthi what's with this?

Comment (abijithanikkuruthi) [reply src/WARCSplitReader.py]: Prof Urbani said he's gonna use only WARC-TREC-ID in one of the QnA sessions and hence the hardcode!

File: src/spark.py (hunk 1 above)
Comment (abijithanikkuruthi) [reply src/spark.py]: Done :+1:

File: src/WARCSplitReader.py (hunk 2 above)
Comment (l8518) [reply src/WARCSplitReader.py]: 👍 Okay, but add this assumption the execution path pls

**Comment by l8518** [comment]
@abijithanikkuruthi merge whenever you are feel its done :-)

**Pull request merged**